<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>937</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simple Painter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/palette.png</normaloff>:/icons/icons/palette.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,10">
    <property name="spacing">
     <number>2</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
      <property name="spacing">
       <number>2</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="ToolBucket" name="tools">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CP::ColorPickerV" name="colorPicker">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50000</width>
          <height>50000</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="StyleSelector" name="styleselector">
        <property name="maximumSize">
         <size>
          <width>78</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="WidthSelector" name="widthselector">
        <property name="maximumSize">
         <size>
          <width>78</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="FillSelector" name="fillselector">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Canvas" name="renderArea" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Image"/>
   <addaction name="actionOpen_Image"/>
   <addaction name="separator"/>
   <addaction name="action_Save"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionFit_To_Screen"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionDebug_info"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/disk-black.png</normaloff>:/icons/icons/disk-black.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/disk.png</normaloff>:/icons/icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;as</string>
   </property>
   <property name="toolTip">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionNew_Image">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/document.png</normaloff>:/icons/icons/document.png</iconset>
   </property>
   <property name="text">
    <string>New Image</string>
   </property>
   <property name="toolTip">
    <string>New Image</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/magnifier-zoom-in.png</normaloff>:/icons/icons/magnifier-zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/magnifier-zoom-out.png</normaloff>:/icons/icons/magnifier-zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionFit_To_Screen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/magnifier-zoom-fit.png</normaloff>:/icons/icons/magnifier-zoom-fit.png</iconset>
   </property>
   <property name="text">
    <string>Fit To Screen</string>
   </property>
   <property name="toolTip">
    <string>Fit To Screen</string>
   </property>
  </action>
  <action name="actionOpen_Image">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/disk.png</normaloff>:/icons/icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/arrow-circle-left.png</normaloff>:/icons/icons/arrow-circle-left.png</iconset>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/arrow-circle.png</normaloff>:/icons/icons/arrow-circle.png</iconset>
   </property>
   <property name="text">
    <string>redo</string>
   </property>
  </action>
  <action name="actionDebug_info">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/gear.png</normaloff>:/icons/icons/gear.png</iconset>
   </property>
   <property name="text">
    <string>debug info</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ToolBucket</class>
   <extends>QFrame</extends>
   <header>toolbucket.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidthSelector</class>
   <extends>QComboBox</extends>
   <header>selectors.h</header>
  </customwidget>
  <customwidget>
   <class>StyleSelector</class>
   <extends>QComboBox</extends>
   <header>selectors.h</header>
  </customwidget>
  <customwidget>
   <class>CP::ColorPickerV</class>
   <extends>QFrame</extends>
   <header>colorpicker.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FillSelector</class>
   <extends>QFrame</extends>
   <header>selectors.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
